app/styles/front.scss

```diff
@@ -637,10 +637,7 @@ $autre-lighter: #cce4e7;
     color: #888;
   }
 
-  .montant::after {
-    @extend .fa;
-    @extend .text-warning;
-    content: $fa-var-warning;
+  i {
```

Comment:
This is a much, much larger selector. And I find it weird that we have to set these properties ourselves, it looks like basic positioning that should be provided by Bootstrap. Can't we get rid of the block altogether?